app/views/unattended/provisioning_templates/PXEGrub2/preseed_default_pxegrub2_autoinstall.erb

```diff
@@ -19,7 +19,13 @@ test_on:
 #   System locale
 #
 <%
-  efi_suffix = 'efi'
+  os_major = @host.operatingsystem.major.to_i
+
+  if (os_major > 12 && os_major < 24)
```

Comment:
Please include an OS name check like in the other file because this will be incorrect for Debian 13.

Reply:
Autoinstall is Ubuntu only, Debian is using preseed.

- https://www.debian.org/releases/stable/amd64/apb.en.html
- https://github.com/theforeman/foreman/pull/10455/files#diff-11373224f6c767c1f0891059e85418cf53dc3a3ffa5cfad088bf2a11d05cc50cR6